src/client.ts

```diff
@@ -604,6 +604,25 @@ export class UXDClient {
     });
   }
 
+  public createSetMangoDepositoryQuoteMintAndRedeemSoftCapInstruction(
+    softCap: number,
+    controller: Controller,
+    depository: MangoDepository,
+    authority: PublicKey,
+    options: ConfirmOptions
+  ): TransactionInstruction {
+    const softCapNativeBN = new BN(
```

Comment:
You don't need to wrap the BN

`const softCapNativeBN = new BN(softCap).mul(new BN(10).pow(new BN(depository.quoteMintDecimals)));`